lib/prediction/predictor.js

```diff
@@ -107,7 +109,7 @@ class Worker extends events.EventEmitter {
         this._requests.clear();
     }
 
-    request(task, context, question) {
+    request(task, context, question, answer=undefined) {
```

Comment:
You don't need `=undefined`, in JS everything is undefined if unspecified.
(And if you want it, put spaces before and after the equal sign)